components/cam/cime_config/testdefs/testmods_dirs/cam/homme/shell_commands

```diff
@@ -0,0 +1 @@
+./xmlchange CAM_TARGET=preqx_kokkos
```

Comment:
Would you please explain what this file does?

Reply:
If you add cam-homme as a testmod, e.g. with a testcase like XXX.GGGG.CCCC.machine_compiler.cam-homme, then your case will have CAM_TARGET set to preqx_kokkos.